File: Web/GuessWord.Backend/GuessWord.Domain/Entities/UserWord.cs
using GuessWord.Domain.Enums;

namespace GuessWord.Domain.Entities
{
    public class UserWord : BaseEntity
    {
        public int UserId { get; set; }
        public int WordId { get; set; }
        public int Complexity { get; set; }
        public WordStatus Status { get; set; }
        public int TargetRepeatNumber { get; set; }
        public int RepeatNumber { get; set; }

        public User User { get; set; }
        public Word Word { get; set; }
    }
}

File: Web/GuessWord.Frontend/src/app/models/userWord.ts
export class UserWord {
    id?: number;
    word: string;
    translations: string[];
    status: number;

    constructor(id: number | undefined, word: string, translations: string[], status: number){

        this.id = id,
        this.status = status,
        this.translations = translations,
        this.word = word
    }
}